Networking/BLIP/BLIPConnection.cc

```diff
@@ -696,4 +707,6 @@ namespace litecore::blip {
 
     websocket::WebSocket* Connection::webSocket() const { return _io->webSocket(); }
 
+    Retained<websocket::WebSocket> Connection::webSocketSafe() const { return _io ? _io->webSocketSafe() : nullptr; }
```

Comment:
Similarly, let's replace the `webSocket()` method with this one.

That will mean also changing `Replicator::webSocket()` to return `Retained<WebSocket>`.